package/yast2-iscsi-lio-server.changes

```diff
@@ -1,6 +1,14 @@
 -------------------------------------------------------------------
 Wed Jul  6 11:00:53 CEST 2016 - gs@suse.de
 
+- fate#318832, Redesign authentication UI
+- A bug fix for discovery session without authentication.
+- Changed online help.
+- 3.1.20
```

Comment:
this is wrong way how to add changes, please use `osc vc` it adds proper timestamp and also you as author of change.